README.md

```diff
@@ -403,7 +403,7 @@ Installs [Helm](https://helm.sh)
 
 * USE_helm: Enable this feature
 * DEBUG_helm: Debug this feature
-* Environment HELM_VERSION: Valid Helm version to install (e.g. 1.5.4)
+* Environment HELM_VERSION*: Valid Helm version to install (e.g. 1.5.4)
```

Comment:
Well, i do have my issues with the asterisk. The documentation is SO big, not many might check the part where it indicates that it means "required". Can't we just plainly write " (required)" instead of "*"?

Reply:
Done